go.mod

```diff
@@ -1,129 +1,138 @@
 module github.com/kubernetes-sigs/aws-efs-csi-driver
 
 require (
-	github.com/aws/aws-sdk-go v1.44.76
-	github.com/container-storage-interface/spec v1.5.0
+	github.com/aws/aws-sdk-go v1.44.116
+	github.com/container-storage-interface/spec v1.7.0
 	github.com/golang/mock v1.6.0
 	github.com/kubernetes-csi/csi-test v1.1.1
 	github.com/mitchellh/go-ps v0.0.0-20170309133038-4fdf99ab2936
 	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
-	google.golang.org/grpc v1.47.0
-	k8s.io/api v0.22.3
-	k8s.io/apimachinery v0.24.3
+	github.com/onsi/gomega v1.23.0
+	google.golang.org/grpc v1.49.0
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
 	k8s.io/client-go v1.5.2
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.22.3
-	k8s.io/mount-utils v0.22.3
+	k8s.io/kubernetes v1.26.1
```

Comment:
Latest supported EKS version is 1.25, so I think we should stick to 1.25

Reply:
Yes, checked in the first part but forgot to close this PR.